modules/indexing/src/test/java/org/apache/ignite/internal/processors/cache/persistence/db/LongDestroyDurableBackgroundTaskTest.java

```diff
@@ -641,6 +641,34 @@ public void testDestroyTaskLifecycle() throws Exception {
         assertTrue(taskLifecycleListener.check());
     }
 
+    /**
+     * Check that index deletion tasks are executed successfully when corresponding cache group is deleted.
+     *
+     * @throws Exception If failed.
+     */
+    @Test
+    public void testIrrelevantTasksAreCleared() throws Exception {
```

Comment:
How does this test ensure that stopping node happens during index removing? What if the index drop happened before the node stopped and this test doesn't check anything.

Reply:
As in other tests, this is achieved by significant delay in org.apache.ignite.internal.processors.cache.persistence.db.LongDestroyDurableBackgroundTaskTest.H2TreeTest#destroyDownPages and size of the index guarantees that this method is called multiple times in one thread. But I can add assertion to ensure that task hadn't completed when grid is stopped.